implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -143,6 +144,13 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "mp.jwt.verify.publickey.location", defaultValue = NONE)
     private Optional<String> mpJwtLocation;
 
+    /**
+     * @since 1.2
+     */
+    @Inject
+    @ConfigProperty(name = Names.VERIFIER_PUBLIC_KEY_ALGORITHM)
+    private Optional<SignatureAlgorithm> mpJwtVerifierPublicKeyAlgorithm;
```

Comment:
I'm not sure using Names helps actually, I'd like to see the name of the property by looking at the source here, besides Names `Verifier` does not match `verify` in `mp.jwt.verify`, I probably used `Names.Verifier` because this pattern was already there. Anyway, may be just lets have `mpJwtPublicKeyAlgorithm` similarly to `mpJwtPubicKeyLocation` to keep it shorter a bit

Reply:
Ok, I can change it to use the plain String. I thought it was better to use the static name so it would be easier for search.

Reply:
Please, it feels it will be easier to read the source. And please have either `mpJwtPublicKeyAlgorithm` or `mpJwtVerifyPublicKeyAlgorithm`, I'd go for the former as we currently don't use `verify` in other similar properties and don't have to match the internal property names one to one with the spec property names...